docs/notebooks/mappings.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mapped modules\n",
+    "\n",
+    "`xclim` leaves users the ability to generate their own modules as one-to-one mappings with the existing indice library. These mappings can help in reproducing existing indice libraries (such as ICCLIM), with the added benefit of CF-compliant metadata, multilingual metadata support, and optimized calculations using federated resources (using Dask). By wrapping functions with different thresholds from default values, any module can leverage the logic of indices to create their own custom module without having to rewrite indices.\n",
+    "\n",
+    "Presently, xclim is capable of approximating the indices developed in ICCLIM (https://icclim.readthedocs.io/en/latest/intro.html) and is open to contributions of new indices and library mappings.\n",
+    "\n",
+    "This notebook serves as an example of how one might go about creating their own library of mapped indices."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll begin by examining how the mapped modules behave"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# importing libraries\n",
+    "\n",
+    "import xarray as xr\n",
+    "xr.set_options(display_style='html')\n",
+    "\n",
+    "import xclim\n",
+    "from xclim import indices\n",
+    "from xclim import icclim\n",
+    "from xclim import subset\n",
+    "import numpy as np\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds = xr.tutorial.open_dataset('air_temperature')\n",
+    "daily_tas = ds.air.resample(time='D').mean(keep_attrs=True)\n",
+    "\n",
+    "fig, axes = plt.subplots(ncols=2, figsize=(12,5))\n",
+    "\n",
+    "daily_tas.sel(lat=45, lon=-71, method=\"nearest\").plot(ax=axes[0], color=\"purple\")\n",
+    "daily_tas.sel(lat=45, lon=-71, method=\"nearest\").plot.hist(ax=axes[1], color=\"green\")\n",
+    "plt.draw"
```

Comment:
I believe you shouldn't need `plt.draw`?

Reply:
Fixed.